fem/sidredatacollection.cpp

```diff
@@ -489,6 +504,73 @@ createMeshBlueprintTopologies(bool hasBP, const std::string& mesh_name)
 }
 
 // private method
+#ifdef MFEM_USE_MPI
+void SidreDataCollection::createMeshBlueprintAdjacencies(bool hasBP)
+{
+   ParMesh *pmesh = dynamic_cast<ParMesh*>(mesh);
+
+   const int GRP_SZ = 35;
+   char group_str[GRP_SZ];
+
+   // TODO(JRC): Separate this out into group hierarchy setup and data allocation
+   // stages like all of the other "createMeshBlueprint*" functions.
```

Comment:
I added a check for the unsupported case that will terminate execution with an error. This should be sufficient for now, and will serve as a reminder as well.